vic/drivers/shared_image/src/vic_init.c

```diff
@@ -986,8 +986,7 @@ vic_init(void)
                 }
                 sum += soil_con[i].AreaFract[j];
             }
-            // TBD: Need better check for equal to 1.
-            if (sum != 1.) {
+            if (!assert_close_double(sum, 1.0, 0., 0.001)) {
```

Comment:
Let's have @tbohn and @bartnijssen weigh in here.

Options are: 

1. Always divide by the sum and only print a warning when the difference is larger than a threshold (0.001).
2. Only divide by the sum when the difference is larger than a very small threshold (1e-20), print a warning in this case.
3. Raise an error in cases where the difference is greater than a very small threshold (1e-20) instead of doing work for the user.

Thoughts?